### ISSUE TITLE
Question, perhaps a bug, on inline source implementation: `global` vs `globals`

### ISSUE THREAD

**Issue by Martii** [eslint/eslint#12128]
**Tell us about your environment**

* **ESLint Version:** Any
* **Node Version:** Any
* **npm Version:** Any

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

n/a and defaults in packages
</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

`/* global someGlobal */`

vs.

`/* globals someGlobal */`

n/a

**What did you expect to happen?**

Trying to develop some consistency between all the linters to code against. I have noticed that ESLint doesn't seem to care on pluralization as do other linters *(could be wrong but this is what some packages, extensions, and add-ons seem to be doing quietly for compatibility)*.

**What actually happened? Please include the actual, raw output from ESLint.**

When using either `global` or `globals` inline comments in source the warning triangles go away in those packages, extensions, and/or add-ons.

Appreciate the clarity _here_ on which one to use for ESLint so we can know what to expect *(i.e. best practice as [the docs](https://eslint.org/docs/user-guide/configuring#specifying-globals) have)*. :smile:

**Comment by platinumazure** [comment]
Hi @Martii, thanks for the issue.

Our [global configuration documentation](https://eslint.org/docs/user-guide/configuring#specifying-globals) recommends the use of `/* global */`. However, I think we support both `/* global */` and `/* globals */` for historical reasons.

**Comment by Martii** [comment]
@platinumazure 

Awesome. Thank you for taking a moment to explain it.


Hmmm seems to do this compatibility also at the [demo site](https://eslint.org/demo) with de/pluralization with:

``` js
switch (someId) {
  case 1:
    // does something
    /* falls through */
  case 2:
    //does something more
    /* falls through */
  default:
    // does something even more
}
```
vs 
``` js
switch (someId) {
  case 1:
    // does something
    /* fall through */
  case 2:
    //does something more
    /* fall through */
  default:
    // does something even more
```

This makes it a bit easier. :+1:

Alright we'll look for more compatibility use cases and adjust accordingly when needed. Thank you again. :smile:
